modules/islandora_xslt_populator/islandora_xslt_populator.module

```diff
@@ -62,6 +62,7 @@ function islandora_xslt_populator_islandora_populator() {
       'output' => array(
```

Comment:
Instead of adding the undo callback, could just add another entry where we're instructed to stash the original, and [it will get handled](https://github.com/Islandora/islandora_populator/blob/11e5cc089334d06cda3dd0f2caa4554ea7c194cc/includes/input.form.inc#L82-L90).